src/csharp/Grpc.Core.Api/Grpc.Core.Api.csproj

```diff
@@ -22,6 +22,10 @@
 
   <Import Project="..\Grpc.Core\SourceLink.csproj.include" />
 
+  <ItemGroup>
```

Comment:
The version info file is generated from a template:
https://github.com/grpc/grpc/blob/master/templates/src/csharp/Grpc.Core/VersionInfo.cs.template - if the file moves, the template needs to move as well (hence the sanity test failure).

Reply:
It might make sense to move the VersionInfo.cs file under Grpc.Core.Api directory, but then all .csproj files including VersionInfo must be updated (there's more of them).

Reply:
Moved